internal/rbd/errors.go

```diff
@@ -42,4 +42,6 @@ var (
 	ErrMissingImageNameInVolID = errors.New("rbd image name information can not be empty in volID")
 	// ErrDecodeClusterIDFromMonsInVolID is returned when mons hash decoding on migration volID.
 	ErrDecodeClusterIDFromMonsInVolID = errors.New("failed to get clusterID from monitors hash in volID")
+	// ErrUnHealthyImage is returned when image is not healthy.
+	ErrUnHealthyImage = errors.New("image is not healthy")
```

Comment:
@Madhu-1 do we need to add `mirrored image...` instead of `image` , so that it clear that, this error is specifically for mirroring enabled one?

Reply:
i can do it.